examples/smart_contracts/v2/python/account_delegation.py

```diff
@@ -20,61 +19,28 @@ def wait_for_confirmation(client, txid):
         txid, txinfo.get('confirmed-round')))
     return txinfo
 
-# Read a file
-def load_resource(res):
-    dir_path = os.path.dirname(os.path.realpath(__file__))
-    path = os.path.join(dir_path, res)
-    with open(path, "rb") as fin:
-        data = fin.read()
-    return data
-
 try:
-
     # Create an algod client
-    # algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
-    # algod_address = "http://localhost:4001"
-    algod_token = "6b3a2ae3896f23be0a1f0cdd083b6d6d046fbeb594a3ce31f2963b717f74ad43"
-    algod_address = "http://127.0.0.1:54746"
-    # algod_token = "<algod-token>"
-    # algod_address = "<algod-address:port>" 
-    # receiver = "<receiver-address>" 
+    algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    algod_address = "http://localhost:4001"
+
     receiver = "ATTR6RUEHHBHXKUHT4GUOYWNBVDV2GJ5FHUWCSFZLHD55EVKZWOWSM7ABQ" 
     algod_client = algod.AlgodClient(algod_token, algod_address)
 
     myprogram = "samplearg.teal"
-    # myprogram = "<filename>"
     # Read TEAL program
-    data = load_resource(myprogram)
-    source = data.decode('utf-8')
+    data = open(myprogram, 'r').read()
     # Compile TEAL program
-    # // This code is meant for learning purposes only
```

Comment:
I prefer to keep the commented TEAL program inline. This one is short enough to read and understand here.

Reply:
I find it redundant and distracting given the file is easily accessible in the same directory and referenced here. But I know this is subjective and I don't feel that strongly about it. I'll add it back in.

Reply:
Another method would be to just add the code as a string assigned to a variable and not reference the separate file at all.